docs/release-notes.rst

```diff
@@ -10,6 +10,39 @@
  Version 0.19
 **************
 
+Version 0.19.8
+==============
+
+**Release Date:** December 02, 2022
+
+**Breaking Changes**
+
+-  REST APIs: ``GetModelVersion``, ``PatchModelVersion``, ``DeleteModelVersion`` APIs now take
+   sequential model version number ``model_version_num`` instead of a surrogate key
+   ``model_version_id``.
```

Comment:
```suggestion
-  API: The ``GetModelVersion``, ``PatchModelVersion``, and ``DeleteModelVersion`` APIs now take
   a sequential model version number ``model_version_num`` instead of a surrogate key
   ``model_version_id``.
```